test/lib/ansible_test/_internal/coverage_util.py

```diff
@@ -69,7 +69,7 @@ class CoverageVersion:
 
 COVERAGE_VERSIONS = (
     # IMPORTANT: Keep this in sync with the ansible-test.txt requirements file.
-    CoverageVersion('6.5.0', 7, (3, 7), (3, 11)),
+    CoverageVersion('7.2.7', 7, (3, 7), (3, 12)),
```

Comment:
Now that 7.3.0 is out, we should use that. Since it drops Python 3.7 support we'll need a new entry here:

```suggestion
    CoverageVersion('7.3.0', 7, (3, 8), (3, 12)),
    CoverageVersion('6.5.0', 7, (3, 7), (3, 7)),
```